ajax/libs/snarkdown/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "snarkdown",
+  "description": "Transform Markdown into HTML.",
+  "authors": [
+    "Jason Miller <jason@developit.ca>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/developit/snarkdown"
+  },
+  "license": "MIT",
+  "keywords": [
+    "markdown",
+    "gfm",
+    "md",
+    "html",
+    "parser"
+  ],
+  "filename": "snarkdown.min.js",
```

Comment:
Let's move `"filename": "snarkdown.min.js",` between `name` and `description` field.
Because we'd like let people know what file they need it in the first glance. 
Thank you.